dmcontent/govuk_frontend.py

```diff
@@ -70,6 +70,11 @@ def from_question(
             "macro_name": "govukInput",
             "params": govuk_input(question, data, errors),
         }
+    if question.type == "list":
```

Comment:
This should be an `elif` (easy mistake to make! I'm trying to think how we could write a test to catch it 🤔 )

```suggestion
    elif question.type == "list":
```

Reply:
Actually I've just realised in this case it doesn't matter as we're returning None either way, but better to be safe than sorry!